modules/core/src/math/matrix/vpMatrix.cpp

```diff
@@ -128,7 +128,7 @@ void compute_pseudo_inverse(const vpMatrix &U, const vpColVector &sv, const vpMa
 
   unsigned int sv_size = sv.size();
   for (unsigned int i = 0; i < sv_size; ++i) {
-    if (sv[i] > (maxsv * svThreshold)) {
+    if (sv[i] >(maxsv * svThreshold)) {
```

Comment:
Should be `if (sv[i] > (maxsv * svThreshold)) {`. Is your visual code well configured ?

Reply:
Each time I save after correcting the missing space, it is erased by VSCode. There must be a problem in the configuration again.